Dockerfile.service_discovery.multi

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+# SPDX-License-Identifier: MIT
+
+ARG RUST_VERSION=1.70
+
+FROM --platform=$BUILDPLATFORM docker.io/library/rust:${RUST_VERSION} AS builder
+
+# Dockerfile for building Eclipse Chariott Service Discovery container
```

Comment:
Consider adding a comment here about the multiple target architectures and maybe which ones are currently supported